fv3gfs-physics/fv3gfs/physics/_config.py

```diff
@@ -97,10 +97,10 @@ class PhysicsConfig:
     tice: float = NamelistDefaults.tice
     alin: float = NamelistDefaults.alin
     clin: float = NamelistDefaults.clin
-    namelist_override: str = None
+    namelist_override: Optional[str] = None
 
     def __post_init__(self):
-        if self.namelist_override is not None:
+        if self.namelist_override:
```

Comment:
nit: same as above

```suggestion
        if self.namelist_override is not None:
```

Reply:
I like seeing 'is not None' when reading the code, but that is just a personal preference, I can change it.